schemas/org.gnome.shell.extensions.gtile.gschema.xml

```diff
@@ -38,133 +38,173 @@
 		      <summary>Preset resize 17.</summary>
 		</key>
 		<key type="as" name="preset-resize-18">
-		      <default><![CDATA[['']]]></default>
+		      <default><![CDATA[['<Super><Control>KP_8']]]></default>
 		      <summary>Preset resize 18.</summary>
 		</key>
 		<key type="as" name="preset-resize-19">
-		      <default><![CDATA[['']]]></default>
+		      <default><![CDATA[['<Super><Control>KP_9']]]></default>
 		      <summary>Preset resize 19.</summary>
 		</key>
 		<key type="as" name="preset-resize-20">
 		      <default><![CDATA[['']]]></default>
 		      <summary>Preset resize 20.</summary>
 		</key>
+		<key type="as" name="preset-resize-21">
+		      <default><![CDATA[['<Super><Shift>KP_1']]]></default>
+		      <summary>Preset resize 21.</summary>
+		</key>
+		<key type="as" name="preset-resize-22">
+		      <default><![CDATA[['<Super><Shift>KP_2']]]></default>
+		      <summary>Preset resize 22.</summary>
+		</key>
+		<key type="as" name="preset-resize-23">
+		      <default><![CDATA[['<Super><Shift>KP_3']]]></default>
+		      <summary>Preset resize 23.</summary>
+		</key>
+		<key type="as" name="preset-resize-24">
+		      <default><![CDATA[['<Super><Shift>KP_4']]]></default>
+		      <summary>Preset resize 24.</summary>
+		</key>
+		<key type="as" name="preset-resize-25">
+		      <default><![CDATA[['<Super><Shift>KP_5']]]></default>
+		      <summary>Preset resize 25.</summary>
+		</key>
+		<key type="as" name="preset-resize-26">
+		      <default><![CDATA[['<Super><Shift>KP_6']]]></default>
+		      <summary>Preset resize 26.</summary>
+		</key>
+		<key type="as" name="preset-resize-27">
+		      <default><![CDATA[['<Super><Shift>KP_7']]]></default>
+		      <summary>Preset resize 27.</summary>
+		</key>
+		<key type="as" name="preset-resize-28">
+		      <default><![CDATA[['<Super><Shift>KP_8']]]></default>
+		      <summary>Preset resize 28.</summary>
+		</key>
+		<key type="as" name="preset-resize-29">
+		      <default><![CDATA[['<Super><Shift>KP_9']]]></default>
+		      <summary>Preset resize 29.</summary>
+		</key>
+		<key type="as" name="preset-resize-30">
+		      <default><![CDATA[['']]]></default>
+		      <summary>Preset resize 30.</summary>
+		</key>
 		
 		<key type = "s" name="resize1">
-		       <default>'2x2 0:0 0:0'</default>
+		       <default>'2x2 0:1 0:1'</default>
   	  	       <summary>Preset resize1 settings (example 6x4 1:2 3:3).</summary>
 		</key>       
 		<key type = "s" name="resize2">
-		       <default>'2x2 0:1 0:1'</default>
+		       <default>'2x2 0:1 1:1'</default>
   	  	       <summary>Preset resize2 settings.</summary>
 		</key>       
 		<key type = "s" name="resize3">
-		       <default>'2x2 1:0 1:0'</default>
+		       <default>'2x2 1:1 1:1'</default>
   	  	       <summary>Preset resize3 settings.</summary>
 		</key>       
 		<key type = "s" name="resize4">
-		       <default>'2x2 1:1 1:1'</default>
+		       <default>'4x4 0:0 0:3'</default>
```

Comment:
Wouldn't make more sense this to be ```4x4 0:0 1:3```? Or better yet just ```2x2 0:0 0:1```? When I was trying the default shortcuts I found it weird that for this group (Super+Ctrl) the grid was suddenly 4x4. It's a bit inconsistent, specially if ```Super+Ctrl+8``` and ```Super+Ctrl+2``` splits the screen in half I would expect ```Super+Ctrl+4``` and ```Super+Ctrl+6``` do the same (but with another orientation).

Reply:
Presets resize 1 - resize 10 are mapped to Super+Alt, not Super+Ctrl
There is a Gnome built-in Super+Left and Super+Right, and it is left half and right half of screen. I see no reason to duplicate Gnome built-in with 2x2 0:1 0:1 as it works just fine and most likely already used by users.
Mapping for Super+Alt 4,5,6 are rather arbitrary, and I am not using them in my own workflow. If you can invent something usable for those key, please post. I will suggest separate commit for preset improvements.
I am using in my own workflow Super+Ctrl 1,4,7 - left thirds, and 3,6,8 - right thirds. I am using 8 for left two thirds, 5 for right two third. 2 is mapped voluntarily and not useful, should be remapped to something.
And, 0 is not mapped for all groups - Super+Alt, Super+Shift, Super+Ctrl. Suggest something!

Reply:
Sorry, not used to Gnome default bindings yet. So in this case I think a good default would be to leave ```resize4``` and ```resize6``` like you suggested, but change ```resize5``` to strech vertically:

```
 		<key type = "s" name="resize5">
		       <default>'4x4 1:0 2:3'</default>
   	  	       <summary>Preset resize5 settings.</summary>
 		</key>  
```

Reply:
That again will duplicate some Gnome functionality...
Gnome have build-in action "maximize vertically" and "maximize horizontally" which are not mapped to any keyboard shortcuts by default.
I do (and I recommend in README and help) to map "Max horiz" to Super+Keypad_6 and "Max vert" to Super+Keypad_8 (my other recommendations for mapping are Super+Keypad_2 for shade and Super+Keypad+4 for Full screen). So, and window that stretch between opposite sides can be achieved by stretching smaller window horizontally or vertically, by Gnome built-in.
gTile idea is to supplement Gnome. And it should do something that Gnome cannot. I am trying to define keyboard presets to get window position/size which is not achievable by one or two keystrokes in Gnome/gTile. Your suggestions are good... but they are duplicate.
It does not mean it should not be implemented. Everybody should try how keyboard presets works, and if new preset does help, it have to be implemented.